definitions/checks/restore/validate_interfaces.rb

```diff
@@ -0,0 +1,24 @@
+require 'foreman_maintain/utils/backup'
+
+module Checks::Restore
+  class ValidateInterfaces < ForemanMaintain::Check
+    metadata do
+      description 'Validate network interfaces match the backup'
+
+      param :backup_dir,
+            'Path to backup directory',
+            :required => true
+      manual_detection
+    end
+
+    def run
+      backup = ForemanMaintain::Utils::Backup.new(@backup_dir)
+      invalid_interfaces = backup.validate_interfaces
+      msg = 'The following features are enabled in the backup, '\
```

Comment:
Before merging I just felt we are not telling where interfaces are missing, and I have added newline as message looks long for a line,
```suggestion
      msg = 'The following features are enabled in the backup, '\
                 "\nbut the system does not have the interfaces used by these features: "
```